pkg/pillar/cmd/nim/controllerdns_test.go

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package nim
+
+import (
+	"flag"
+	"net"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/agentbase"
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/dpcmanager"
+	"github.com/lf-edge/eve/pkg/pillar/netmonitor"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/sirupsen/logrus"
+)
+
+// TODO: remove?
+var testDomains = flag.String("domains", "", "comma separated list of domains to resolve")
```

Comment:
> TODO: remove?

If not used then yes. If someone needs to test resolution of some specific domains, then they can add or modify existing UT.